earthpy/plot.py

```diff
@@ -123,17 +123,23 @@ def plot_bands(
         "Input arr should be a numpy array"
 
     if title:
+        if isinstance(title, str):
+            title = [title]
+
+        # A 2-dim array should only be passed one title
         if (arr.ndim == 2) and (len(title) > 1):
```

Comment:
We could simplify this line to 
```suggestion
        if arr.ndim == 2 and len(title) > 1:
```